docs/tempo/website/metrics-generator/app-performance-mgmt.md

```diff
@@ -0,0 +1,167 @@
+
+If you are using the metrics-generator, then it processes traces and generates service graphs in the form of time series metrics like:
+
+```yaml
+tempo_service_graph_request_total{client="app", server="db"} 20
+```
+
+For information about service graphs and how they are calculated, refer to the [Service Graphs documentation]({{< relref "../metrics-generator/service_graphs.md" >}}).
+
+## Use filters to reveal details
+
+The APM dashboard provides a gateway to your tracing information in the service graphs and span metrics. 
+his dashboard is derived from a fixed set of metrics queries.
+These underlying queries can not be changed.
+However, you can choose which traces are included in the metrics query by filtering.
+
+You can explore data by clicking on selectable items or by using filters.
+
+### Selecting items or nodes for more detail
+
+Clicking on selectable items, such as underlined text in the APM table or nodes on the service graph, lets you reveal specific details based upon your selection.
+
+In the APM table, you can select items in the **Rate**, **Error Rate**, **Duration (p90)**, and **Links** columns. Choosing one if these items provides details about the span metrics.
+
+<p align="center"><img src="../apm-rate-drilldown.png" alt="APM table with rate drill-down"></p>
+
+
+You can view request rate, request histogram, failed request rate, and traces for any node in the service graph.
+To view more information, select the node in the service graph and then choose an option from the popup.
+For details on navigating the service graph, refer to the [Node graph panel](https://grafana.com/docs/grafana/next/visualizations/node-graph/) documentation.
+
+<p align="center"><img src="../apm-service-graph-drilldown.png" alt="APM service graph with drill-down"></p>
+
+
+### Filter with metric queries
+
+Using the filters at the top of the screen, you can narrow the data set based upon span attributes (key-value pairs or labels).
+The filters build a query to refine what is shown in the service graph and span metrics.
+You can add one or more label filters.
+
+To use the filters:
+
+1. At the top of the Service Graph, select the text box after **Filter** to display a list of available labels. In this case, **server** is selected. <br /><img src="../apm-query-filter-label.png" alt="APM service graph with grid layout">
+
+1. Select or search for a value for the label. In this case, the value of **server** is equal to **tempo-ingester**. The default operator is equals (=). <br /> <img src="../apm-query-filter-value1.png" alt="APM select value for label">
+
+1. Optional: Change the operator by selecting **=** and choosing a new option from the drop-down. <br /><img src="../apm-filter-operator.png" alt="APM filter operators">
+
+
+1. Optional: Add additional key-value pairs to refine the data set. Any subsequent label filters use AND, which requires both key-value pairs to be presents for matches.
```

Comment:
I guess that means "OR" is not supported?

Reply:
When you look at the screenshot, I did not find a way to change the AND statement to an OR.

Reply:
@joey-grafana Is there another way to change the AND?

Reply:
It's just AND at them moment.

Reply:
Thank you.